kanopi/rooseveltedu
# 26294738 bulk image

## Description
Provide a method for initial upload of images and files for Drupal to manage.

## Related Tickets

* [Bulk Image Import](https://kanopi.teamwork.com/#/tasks/26294738)

## Steps to Validate
1. Go to `/admin/content/media`
2. Observe the **Upload Media in bulk** button
3. Observe hundreds of Media items
4. Go to `/admin/content/browser`
5. See the Media Browser with all the files organized by directory
6. Click through the directory tree and see the Media items
